profile/README.md

```diff
@@ -0,0 +1,31 @@
+
+![Oracle Logo](ogho_banner.png "Open source at Oracle. Everyone is invited. Let's collaborate together")
+
+# Open Source at Oracle
+
+Oracle is the purveyor of the industry’s most widely adopted open source technologies, such as [Java](https://openjdk.java.net/) and [MySQL](https://www.mysql.com/), and one of the founding members of the [Linux Foundation](https://linuxfoundation.org/), the [Eclipse Foundation](https://www.eclipse.org/), and the [Java Community Process](https://jcp.org/en/home/index).
+
+But what has guided us along the way?
+
+Our mission is to empower developers and continue supporting open source technologies. Through Oracle Cloud Infrastructure and its 37 cloud regions, we’re providing more opportunities than ever to make open source technology available to everyone.<br><br>
+
+## Get Started
+
+[Learn about open source at Oracle](https://developer.oracle.com/open-source/)
+
+[Join Oracle's developer community](https://developer.oracle.com/)<br><br>
+
+## All Oracle GitHub organizations
+
+* [Oracle on GitHub](https://github.com/oracle)
+* [AddThis](https://github.com/addthis)
+* [Fn Project](https://github.com/fnproject)
+* [Java Community Process](https://github.com/jcp-org)
+* [Java EE](https://github.com/javaee)
+* [MySQL](https://github.com/mysql)
+* [Oracle DevRel](https://github.com/oracle-devrel)
+* [Oracle GitHub Actions](https://github.com/oracle-actions)
+* [Oracle LiveLabs](https://github.com/oracle-livelabs)
```

Comment:
This org isn't ready yet, so can we not list it?

Reply:
ok